requirements.txt

```diff
@@ -1,5 +1,4 @@
 numpy
-opencv-python
 Pillow
 torch>=1.7
 torchvision>=0.8.0
```

Comment:
And I think we can delete this file entirely 'cause we already use `pyproject.toml` for it